bokeh/models/annotations.py

```diff
@@ -321,6 +326,14 @@ class Legend(Annotation):
             LegendItem(label="3*sin(x)" , renderers=[r3, r4])
         ])
 
+    Or, when constructing a legend according to scatter glyph size:
+
+        legend = Legend(items=[
```

Comment:
needs a `.. code-block:: python` like the other blocks above and below (and make sure to match vertical whitespace, sphinx is picky...)